daprdocs/content/en/developing-applications/building-blocks/state-management/howto-outbox.md

```diff
@@ -106,6 +108,73 @@ spec:
     value: "newOrder"
 ```
 
+### Shape the outbox pattern message
+
+You can override the outbox pattern message saved to the database during the transaction by setting a different message. This is done via a projected transaction payload, which is ignored in the database, but used as the outbox pattern message published to the user topic.
+
+In the following Go example of a state transaction, the value of `"2"` is saved to the database, but the value of `"3"` is published to the end-user topic.
+
+```go
```

Comment:
If we can add tabs for multiple samples where one of them is a raw HTTP that would be very helpful